cicd/3-app/aiproxy/template.yml

```diff
@@ -5,6 +5,9 @@ Description: Provision an instance of the AI Proxy service.
 # Dependencies: This template has dependencies, look for !ImportValue in the Resources section.
 
 Parameters:
+  EnvironmentType:
+    Type: String
+    Description: Environment type (e.g. 'development', 'staging', 'test', 'prod').
```

Comment:
is this indicative of the environment types we plan to support for aiproxy?

Reply:
Moreso just examples to provide context. Right now we have a production environment and a non-production environment (named 'test'). We can definitely add another environment ('staging') if we would like to maintain a horizontal slice between `code-dot-org` and `aiproxy`, or we could simply point all non-prod environments of code-dot-org at the 'aiproxy-test.code.org' environment.